packages/graphql/src/utils/execute.ts

```diff
@@ -53,7 +53,7 @@ async function execute(input: {
     }
 
     // @ts-ignore: Required to set connection user agent
-    input.context.driver._userAgent = `${environment.NPM_PACKAGE_VERSION}/${environment.NPM_PACKAGE_NAME}`; // eslint-disable-line no-underscore-dangle
+    input.context.driver._userAgent = `${environment.NPM_PACKAGE_NAME}/${environment.NPM_PACKAGE_VERSION}`; // eslint-disable-line no-underscore-dangle
```

Comment:
These are internal API:s and I don't think the driver supports updating the config once the driver is instantiated.
To get this to work with driver `4.3.0` I have to change this to

```js
input.context.driver._config.userAgent = `${environment.NPM_PACKAGE_NAME}/${environment.NPM_PACKAGE_VERSION}`;
```

which implies that it could have been a change in the internal API:s for the 4.3.0 version of the driver.

Reply:
Cool, this works for me now. I added both support for 4.2 and 4.3